src/components/BounceDetailsContainer/index.jsx

```diff
@@ -92,14 +94,26 @@ const BounceRuleDetailed = ({
           <Column className="details-button-column" width={1} offset={11}>
             <span>
               <Button
-                onClick={handleEditClicked}
+                onClick={
+                  editText === "FREE" || editText === "ALREADY"
+                    ? handleEditClicked
+                    : () => {}
+                }
                 id="isEditClicked"
-                onKeyDown={handleEditClicked}
+                onKeyDown={
+                  editText === "FREE" || editText === "ALREADY"
+                    ? handleEditClicked
+                    : () => {}
+                }
                 {...WriteSelectors.editButton}
                 className="sg-button edit-button"
                 type="primary"
+                icon={editText !== "FREE" ? "locked" : ""}
+                disabled={editText !== "FREE"}
               >
-                Edit Rule
+                {editText === "FREE" || editText === "ALREADY"
```

Comment:
For this case we can assign to a variable as well to make it more readable and say like if userCanEditRule so it reads like {userCanEditRule ? "Edit Rule" : "In Use"} and ditto what Dana said about assigning "FREE" "ALREADY" into const variables to edit it all in one place if you need to refactor with a descriptive name of what it means/comments. "ALREADY" is a bit confusing and it would help to see what the third value is in a comment i.e. "IN USE" since we're checking if it's FREE or ALREADY but there is a third unknown value we don't need. const WS_RULE_FREE = "FREE", WS_RULE_etc. "FREE", "ALREADY" can be interpreted differently and some documentation and reassigning to variables will help with that